Formula/mariadb-connector-odbc.rb

```diff
@@ -13,7 +13,7 @@ class MariadbConnectorOdbc < Formula
   def install
     system "cmake", ".", "-DWITH_UNIXODBC=1", "-DMARIADB_FOUND=1",
             "-DWITH_OPENSSL=1",
-            "-DOPENSSL_INCLUDE_DIR=/usr/local/opt/openssl/include/",
+            "-DOPENSSL_INCLUDE_DIR=#{Formula["openssl"].include}",
```

Comment:
currently this is pedantry as `include` == `opt_include` for formulae other than the one at hand, but we still prefer to do `opt_include` to make it explicit